synapse/lib/layer.py

```diff
@@ -1178,6 +1251,13 @@ async def _initLayerStorage(self):
 
         self.nodeeditlog = self.nodeeditctor(self.nodeeditslab, 'nodeedits')
 
+        self.layrslab.on('commit', self._onLayrSlabCommit)
+
+        self.layrvers = self.meta.get('version', 2)
```

Comment:
shouldn't this be 3 ?

Reply:
nope

Reply:
if un-specified we have to assume it's a v2 layer being converted to v3